src/main/java/lotto/controller/LottoController.java

```diff
@@ -19,12 +21,15 @@ public LottoController() {
 
     public void start() {
         LottoPurchaseBudget budget = getBudget();
-        List<LottoTicket> tickets = seller.generateTickets(budget);
+        int manualLottoCount = getManualLottoCount(budget);
 
+        List<LottoTicket> tickets = getManualLottoTickets(manualLottoCount, budget);
+        tickets.addAll(seller.purchaseAutoLotto(budget));
```

Comment:
`List<LottoTicket>`을 가진 **일급 컬렉션**을 만들어보면 어떨까요? 🙂


또한 자동 로또 티켓들 + 수동 로또 티켓들을 각각 구한 뒤, 두 일급 컬렉션을 합하는 것처럼 아래와 같이 변경해보면 조금 더 깔끔해질 것 같네요. 😉

```suggestion
        LottoTickets manualLottoTickets = getManualLottoTickets(manualLottoCount, budget);
        LottoTickets autoLottoTickets = seller.purchaseAutoLottos(budget)
        LottoTickets lottoTickets = manualLottoTickets.addAll(autoLottoTickets);
```


혹은 위의 자동 로또 티켓들과 수동 로또 티켓들을 각각 만들고 더해서 (총) 로또 티켓들을 만드는 로직은 메서드로 분리하는 것도 괜찮은 것 같아요. 😁